src/biogeophys/TemperatureType.F90

```diff
@@ -453,13 +453,17 @@ subroutine InitHistory(this, bounds, is_simple_buildtemp, is_prog_buildtemp )
     call hist_addfld1d (fname='T10', units='K',  &
          avgflag='A', long_name='10-day running mean of 2-m temperature', &
          ptr_patch=this%t_a10_patch, default='inactive')
-
-    if (use_cn .and.  use_crop )then
-       this%t_a5min_patch(begp:endp) = spval
-       call hist_addfld1d (fname='A5TMIN', units='K',  &
-            avgflag='A', long_name='5-day running mean of min 2-m temperature', &
-            ptr_patch=this%t_a5min_patch, default='inactive')
-    end if
+    
+    this%soila10_patch(begp:endp) = spval
+    call hist_addfld1d (fname='SOIL10', units='K',  &
+         avgflag='A', long_name='10-day running mean of 3rd layer soil', &
+         ptr_patch=this%soila10_patch, default='inactive')
+    !if (use_cn .and.  use_crop )then
```

Comment:
Please remove this commented-out line of code and the commented-out `end if` a few lines down.